vendor/bat-native-ledger/src/bat/ledger/internal/static_values.h

```diff
@@ -25,11 +25,11 @@
 "https://balance.rewards.brave.software"
 
 #define PUBLISHER_STAGING_SERVER     \
-"https://publishers-staging-distro.basicattentiontoken.org"
+"https://rewards-stg.bravesoftware.com"
 #define PUBLISHER_PRODUCTION_SERVER  \
-"https://publishers-distro.basicattentiontoken.org"
+"https://rewards.brave.com"
 #define PUBLISHER_DEVELOPMENT_SERVER  \
-"https://creators-distro.brave.software"
+"https://rewards-stg.bravesoftware.com"
```

Comment:
This still needs to be updated. cc @yachtcaptain23

Reply:
what is status on this one @zenparsing @yachtcaptain23 ?

Reply:
Unless I'm mistaken, this is already implemented server-side and client-side. Recommend resolving.

Reply:
@yachtcaptain23 Just to confirm again - we're going to use the same prefix list URL for both staging and development, correct?

Reply:
not sure how we can use the same list if publishers are different